pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/rule/codestyle/AbstractRegexNamingConventionsRule.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.apex.rule.codestyle;
+
+import static net.sourceforge.pmd.properties.PropertyFactory.regexProperty;
+
+import java.util.Map;
+import java.util.regex.Pattern;
+
+import net.sourceforge.pmd.lang.apex.ast.ApexNode;
+import net.sourceforge.pmd.lang.apex.rule.AbstractApexRule;
+import net.sourceforge.pmd.properties.PropertyBuilder;
+import net.sourceforge.pmd.properties.PropertyDescriptor;
+
+public abstract class AbstractRegexNamingConventionsRule extends AbstractApexRule {
```

Comment:
This class could be package private. It's quite specific to this particular set of rules and wouldn't be of much use outside of PMD.

Reply:
Fixed